dataactvalidator/validation_handlers/validationManager.py

```diff
@@ -229,13 +233,19 @@ def runValidation(self, jobId, interfaces):
                             errorInterface.setRowErrorsPresent(jobId, True)
                         continue
                     valid, failures = Validator.validate(record,rules,csvSchema,fileType,interfaces)
-                    if(valid) :
+                    if valid:
                         try:
-                            tableObject.insert(record,fileType)
+                            record["job_id"] = jobId
+                            record["submission_id"] = submissionId
+                            # temporary fix b/c we can't use '+4' as a column alias :(
+                            if "primaryplaceofperformancezip+4" in record:
+                                record["primaryplaceofperformancezipplus4"] = record["primaryplaceofperformancezip+4"]
```

Comment:
This is very temporary, since the schema team has already provided a machine-friendly list of field names, which will replace the aliases currently used in the models. Knowing this list was imminent, I put this line in as the quickest patch.

It is not my proudest coding moment, to be sure.